static/js/util/api.js

```diff
@@ -33,6 +34,50 @@ export function csrfSafeMethod(method: string): boolean {
   return /^(GET|HEAD|OPTIONS|TRACE)$/.test(method);
 }
 
+const headers = R.merge({ headers: {} });
+
+const method = R.merge({ method: 'GET' });
+
+const credentials = R.merge({ credentials: 'same-origin' });
+
+const setWith = R.curry((path, valFunc, obj) => (
+  R.set(path, valFunc(), obj)
+));
```

Comment:
`getCookie` needs to be called at runtime. Just using `R.set` means that `getCookie` gets called whenever the module is loaded, which is before we can set `document.cookie` in our testing code.